lib/loader.js

```diff
@@ -101,7 +101,9 @@ module.exports = async function (content, sourceMap) {
       compiler.compile(component.template.content, {
         modules: [{
           postTransformNode: node => {
-            node.attrsList.forEach(({ name }) => attrs.add(name))
+            if ("directives" in node) {
+              node.directives.forEach(({ name }) => attrs.add(name))
```

Comment:
Hmm, technically it's a breaking change, if someone is using vuetify-loader to import something based on normal attributes it won't work anyway

We could keep the "attrs" mode as it was before (but without default matcher, still might be a breaking change but at least people will have possibility to do what they want with normal attrs) and add a directive matcher as a separate thing. Thoughts?

Reply:
I don't think that's necessary. If someone requests it sure, but this option isn't documented and I doubt very many people are even using the one that is.

Reply:
Also "something" would have to be a directive anyway because it's added to `component.options.directives`